trikot-viewmodels-declarative-flow/swift/swiftui/Components/VMDTextField.swift

```diff
@@ -28,12 +28,14 @@ public struct VMDTextField<Label>: View where Label: View {
         self.observableViewModel = viewModel.asObservable()
         self.labelBuilder = nil
         self.onFocusChange = onFocusChange
+        self.text = viewModel.text
```

Comment:
This is a new Xcode 14 syntax. Since Xcode 13 is not 100% dead, maybe we could keep the old way for now:
`_text = State(initialValue: viewModel.text)`